modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemViews/ItemInfoView.js

```diff
@@ -19,15 +21,15 @@ function ItemInfoViewOptions({options}) {
 	);
 }
 
-function ItemInfoViewBundle({childItems}) {
+function ItemInfoViewBundle({childItems, options}) {
 	const [expanded, setExpanded] = useState(false);
 
-	return Liferay.FeatureFlags['COMMERCE-9599'] ? (
+	return Liferay.FeatureFlags['COMMERCE-9599'] && options.length > 1 ? (
```

Comment:
Hi @brianikim , is it correct that if you have a single option you fallback to the "old version"?

Reply:
let's add this to the question to ask to the product in the desk check, otherwise this review cycle will never end.